<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapLayerChoice</class>
 <widget class="QWidget" name="MapLayerChoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Layer Choice</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QWidget" name="m_widget" native="true">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>280</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
